app/src/main/java/com/example/gabriel/sociala/PostPhotoActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.gabriel.sociala;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+
+import com.example.gabriel.sociala.R;
+
+public class PostPhotoActivity extends AppCompatActivity {
+
+    ImageView selectedImageView;
+    ImageButton backButton;
+    Button postButton;
```

Comment:
Can we add the profile_img/user_name here too? 
Also are we getting the profile_img via signup?? 

I think it'll be good enough to write 

```
TextView userName; 
ImageView profilePic;
```

Reply:
oops totally forgot about user name.. thanks for pointing out!

Reply:
> Can we add the profile_img/user_name here too?
> Also are we getting the profile_img via signup??
> 
> I think it'll be good enough to write
> 
> ```
> TextView userName; 
> ImageView profilePic;
> ```
oops totally forgot about user name.. thanks for pointing out!